gulpfile.ts

```diff
@@ -161,6 +160,23 @@ task('load-trading-view', (done) => {
             });
             taskHash.html.push(`html-${taskPostfix}`);
 
+            if (buildName === 'desktop') {
+                task(`electron-create-package-json-${taskPostfix}`, [`html-${taskPostfix}`], function (done) {
```

Comment:
String `\`electron-create-package-json-${taskPostfix}\`` is duplicated on line 177.